data/data-pipeline/data_pipeline/etl/sources/ncld_nature_deprived/etl.py

```diff
@@ -0,0 +1,75 @@
+# pylint: disable=unsubscriptable-object
+# pylint: disable=unsupported-assignment-operation
+
+import pandas as pd
+from data_pipeline.config import settings
+
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class NatureDeprivedETL(ExtractTransformLoad):
+    """ETL class for the Nature Deprived Communities dataset"""
+
+    NAME = "ncld_nature_deprived"
+    SOURCE_URL = (
+        settings.AWS_JUSTICE40_DATASOURCES_URL
+        + "/usa_conus_nat_dep__compiled_by_TPL.csv.zip"
+    )
+    GEO_LEVEL = ValidGeoLevel.CENSUS_TRACT
+
+    # Output score variables (values set on datasets.yml) for linting purposes
+    ELIGIBLE_FOR_NATURE_DEPRIVED_FIELD_NAME: str
+    TRACT_PERCENT_IMPERVIOUS_FIELD_NAME: str
+    TRACT_PERCENT_NON_NATURAL_FIELD_NAME: str
+    TRACT_PERCENT_CROPLAND_FIELD_NAME: str
+
+    def __init__(self):
+        # define the full path for the input CSV file
+        self.INPUT_CSV = (
+            self.get_tmp_path() / "usa_conus_nat_dep__compiled_by_TPL.csv"
+        )
+
+        # this is the main dataframe
+        self.df: pd.DataFrame
+
+        # Start dataset-specific vars here
+        self.PERCENT_NATURAL_FIELD_NAME = "PctNatural"
+        self.PERCENT_IMPERVIOUS_FIELD_NAME = "PctImperv"
+        self.PERCENT_CROPLAND_FIELD_NAME = "PctCrops"
+        self.TRACT_ACRES_FIELD_NAME = "TractAcres"
+        # In order to ensure that tracts with very small Acreage, we want to create an eligibility criterion
```

Comment:
Please see this comment for additional explanation

Reply:
This is a helpful comment!